client/src/util/quantile.ts

```diff
@@ -12,17 +12,19 @@ Arguments:
 
 */
 
+import { TypedArray } from "../common/types/entities";
 import { sortArray } from "./typedCrossfilter/sort";
 
-// @ts-expect-error ts-migrate(7006) FIXME: Parameter 'quantArr' implicitly has an 'any' type.
-// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types --- FIXME: disabled temporarily on migrate to TS.
-export default function quantile(quantArr, tarr, sorted = false) {
+export default function quantile(
+  quantArr: Array<number>,
+  tarr: TypedArray,
+  sorted = false
+): Array<number> {
   /*
 	start with the naive (sort) implementation.  Later, use a faster partition
 	*/
-  const arr = sorted ? tarr : sortArray(new tarr.constructor(tarr)); // copy
```

Comment:
Might be wrong, but new keyword wasn't needed

Reply:
it is wrong - doesn't do the same thing (won't make a copy)

You want:
```
const arr = sorted ? tarr : sortArray(new (tarr.constructor as TypedArrayConstructor)(tarr));
```